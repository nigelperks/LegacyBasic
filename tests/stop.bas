10 PRINT "ONE"
20 PRINT "TWO"
30 STOP
40 PRINT "THREE"
