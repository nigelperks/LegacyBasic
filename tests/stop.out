ONE
TWO
30 STOP
Stopped
